src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1142,7 +1158,9 @@ void Graph::Allocate() {
     const auto& edges = allocationContext.edges;
     InitEdgeStatus(edges);
 
-    auto [solution, edgeClusters, m_outputNodesMemBlocks] =
+    MemoryControl::MemorySolution solution;
+    EdgeClusters edgeClusters;
+    std::tie(solution, edgeClusters, m_outputNodesMemBlocks) =
```

Comment:
@maxnick FYI
This change actually fixes regression introduced by the previous PR, because structured binding actually creates a new m_outputNodesMemBlocks variable which shadows the original member variable. So, member variable was not actually updated.